include/solver.hpp

```diff
@@ -1,27 +1,27 @@
 /*
- * GHOST (General meta-Heuristic Optimization Solving Tool) is a C++ library 
```

Comment:
Arf, we can't see changes here because of space conversion indeed. What did you modify in solver.hpp?